examples/inprocess_qtconsole.py

```diff
@@ -0,0 +1,37 @@
+"""
+An example of opening up an RichJupyterWidget in a PyQT Application, this can
+execute either stand-alone or by importing this file and calling
+inprocess_qtconsole.show().
+
+Based on the earlier example in the IPython repository, this has
+been updated to use qtconsole.
+"""
+
+
+from PyQt4 import QtGui
```

Comment:
You should import from `qtconsole` shim instead of doing a PyQt4 import. The thing is `qtconsole` works with PyQt4, PyQt5 and PySide.